renderdoc/driver/d3d11/d3d11_analyse.cpp

```diff
@@ -2607,7 +2605,7 @@ byte *D3D11DebugManager::GetTextureData(ResourceId id, uint32_t arrayIdx, uint32
     if(mip >= mips || arrayIdx >= desc.ArraySize)
       return NULL;
 
-    if(forceRGBA8unorm)
+    if(params.remap)
```

Comment:
In each of the drivers when you're checking remap, can you do `RDCASSERT(params.remap == RGBA8)` ? Since the other cases are not currently implemented.